packages/common/src/HardhatConfig.js

```diff
@@ -1,4 +1,6 @@
 const { getNodeUrl, mnemonic } = require("./TruffleConfig");
+const path = require("path");
+const coreWkdir = path.dirname(require.resolve("@uma/core/package.json"));
```

Comment:
@mrice32 did you figure out a better way to get this path without having to import `@uma/core`?

Reply:
We can do it the same way we do it in TruffleConfig. The caller can pass `workingDirectory` into `getHardhatConfig`.

Thoughts?

Reply:
Done